### DISCUSSION TITLE
Slider hitcircle not immediately disappearing in lazer

### DISCUSSION THREAD

**Discussion by vicodindeok** [ppy/osu#19841] [Q&A]
### Type

Other

### Bug description

In stable, when you don't click and hold on a slider, the hitcircle disappears immediately, but in lazer the hitcircle fades away and can be confusing at times as shown in the example.

### Screenshots or videos

![lazerbug](https://user-images.githubusercontent.com/104724437/185400745-df88dc86-270a-472f-9dfa-019822d6b6a9.gif)

### Version

2022.816.0-lazer

### Logs

[legacy-ipc.log](https://github.com/ppy/osu/files/9373890/legacy-ipc.log)
[performance.log](https://github.com/ppy/osu/files/9373891/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9373892/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9373894/updater.log)
[database.log](https://github.com/ppy/osu/files/9373895/database.log)

**Comment by peppy** [comment]
Lazer is showing it for as long as it is still within the hittable range. Is this an issue to you?

**Reply by LO-oE** [reply]
In my opinion it is a bit distracting from gameplay. Is it possible to add an option to turn this off?

**Reply by peppy** [reply]
The image here shows it only happen when you *miss*. Is that the scenario that it is distracting you in? As I mention above, it's still hittable at this point.

**Reply by LO-oE** [reply]
I know it only happens when you miss and for some people this might be helpful, but my brain is used to search for hitcircles as they appear and it's distracting to see them moving around. Maybe you can get used to it, but at the moment it's more distracting than helpful for me.
